### PR TITLE
Revert "Test source repo sync update trigger"

### PR DESCRIPTION
**Pull request by another-rex**
Reverts google/osv.dev#1972

### CODE REVIEW

**Review by andrewpollock — APPROVED** [review]

File: source_test.yaml [hunk 1]
```diff
@@ -266,16 +266,3 @@
   key_path: 'OSV'
   repo_username: 'git'
 
-- name: 'test'
-  versions_from_repo: True
-  type: 0
-  ignore_patterns: ['allowlist.json']
-  repo_url: 'blah'
```
Comment (andrewpollock) [inline source_test.yaml:273]: I'm assuming this is the culprit for a recent spate of

`ValueError: Source repository link must end with /` that just showed up in staging...

Comment (jess-lowe) [reply source_test.yaml:273]: Yup. Sorry about that 😂

**Review by jess-lowe — APPROVED** [review]
LGTM

File: source_test.yaml (hunk 1 above)
Comment (andrewpollock) [reply source_test.yaml:273]: Potential cherry-on-top here would be to add some validation so that invalidity doesn't get checked in :-)

Comment (jess-lowe) [reply source_test.yaml:273]: Just added a pr to merge to this that should be able to fix that :)

**Pull request merged**